### PR TITLE
Internal: Added tabs and social icons missing E2E tests [ED-11064]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X ] Other... Please describe: playwright tests

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/sanity/includes/widgets/icon.test.js [hunk 1]
```diff
@@ -1,29 +1,37 @@
 const { test, expect } = require( '@playwright/test' );
 const WpAdminPage = require( '../../../pages/wp-admin-page.js' );
+const EditorPage = require( '../../../pages/editor-page' );
+import Content from '../../../pages/elementor-panel-tabs/content';
+import EditorSelectors from '../../../selectors/editor-selectors.js';
 
-test( 'Enable SVG fit-to-size', async ( { page }, testInfo ) => {
+test.beforeAll( async ( { browser }, testInfo ) => {
```
Comment (yotamselementor) [inline tests/playwright/sanity/includes/widgets/icon.test.js:7]: Can you wrap these in a `describe`?

File: tests/playwright/sanity/includes/widgets/tabs.test.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
-const { test, expect } = require( '@playwright/test' );
-const WpAdminPage = require( '../../../pages/wp-admin-page.js' );
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page.js';
+const EditorPage = require( '../../../pages/editor-page.js' );
+import Content from '../../../pages/elementor-panel-tabs/content.js';
```
Comment (yotamselementor) [inline tests/playwright/sanity/includes/widgets/tabs.test.js:4]: Can you wrap these tests in a `describe`

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: tests/playwright/pages/elementor-panel-tabs/content.js [hunk 3]
```diff
@@ -18,22 +19,28 @@ export default class Content {
...
 		await urlInput.clear();
 		await urlInput.type( link );
-		const wheel = this.page.locator( EditorSelectors.button.linkOptions );
+
+		const wheel = this.page.locator( EditorSelectors.button.linkOptions ).first();
 		if ( await wheel.isVisible() ) {
 			await wheel.click();
 		}
+
+		if ( options.targetBlank ) {
```
[the remark this answers is not in the document]
Comment (dmaziar) [reply tests/playwright/pages/elementor-panel-tabs/content.js:31]: it was just a copy-paste error. It should be `if ( options.noFollow ) { `.
Good catch! Fixed.

File: tests/playwright/sanity/includes/widgets/icon.test.js (hunk 1 above)
Comment (dmaziar) [reply tests/playwright/sanity/includes/widgets/icon.test.js]: fixed

File: tests/playwright/sanity/includes/widgets/tabs.test.js (hunk 2 above)
Comment (dmaziar) [reply tests/playwright/sanity/includes/widgets/tabs.test.js:4]: fixed

**Review by yotamselementor — DISMISSED** [review]

File: tests/playwright/sanity/includes/widgets/icon.test.js [hunk 4]
```diff
@@ -1,55 +1,73 @@
...
+		} );
 
-		await page.getByRole( 'button', { name: 'Style' } ).click();
-		await page.locator( '.elementor-switch-label' ).click();
-		await page.getByRole( 'spinbutton', { name: 'Size' } ).fill( '300' );
-	} );
+		await test.step( 'Act', async () => {
+			await contentTab.uploadSVG();
+			await page.getByRole( 'button', { name: 'Style' } ).click();
+			await page.locator( '.elementor-switch-label' ).click();
```
Comment (Barcov) [inline tests/playwright/sanity/includes/widgets/icon.test.js:37]: @dmaziar I'd suggest using a locator from .elementor-control-size to make this less flaky. .elementor-switch-label was not found when I tested this.

**Review by Barcov — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/includes/widgets/icon.test.js (hunk 4 above)
Comment (dmaziar) [reply tests/playwright/sanity/includes/widgets/icon.test.js:37]: the problem was in too fast await contentTab.uploadSVG();.
Should be fine now.

**Review by Barcov — APPROVED** [review]

**Pull request merged**